lib/system-nixpkgs-map.nix

```diff
@@ -113,6 +113,7 @@ in
   GeoIP = [ geoip ];
   pulse-simple = [ libpulseaudio ];
   oath = [ liboauth ];
+  GLEW = [ glew ];
```

Comment:
Recommend moving this up with `GL` and `GLU`.

Reply:
done, but unfortunately build is still failing, see https://pastebin.com/tFRQYYwE and https://github.com/prolic/futr/pull/5